live-examples/js-examples/string/string-slice.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">var str = 'The quick red fox jumped over the lazy dog\'s back.';
```

Comment:
It would be good to have a blank line after this, for readability.

Reply:
Done